app/models/extract_data.py

```
@@ -3,6 +3,7 @@
 """
 
 from pydantic import BaseModel, Field, ConfigDict
+from sqlalchemy import Index
```

Pylint (3.3.7)

[error] Unable to import 'sqlalchemy'

(E0401)